play-media-native/README.md

```diff
@@ -79,3 +79,26 @@ On your phone:
         - `npm run android`
     2. Type the "a" keyboard key to tell Expo to start the application on your Android emulator.
         - The Expo development server will install Expo Go on your Android emulator, compile the application, send it to your Android emulator, and your Android emulator will open the application.
+
+### Generating an APK for Android
```

Comment:
The APK sections should not be inside of the "getting started" section. They should be inside a "Android APK" level 2 heading maybe.